app/controllers/api/v0/search_controller.rb

```diff
@@ -78,6 +78,8 @@ def search
     response.overall_took = ((Time.now - started_at)*1000).round
 
     render json: response, status: :ok
+  rescue Elasticsearch::Transport::Transport::Errors::NotFound
+    render json: 'The specified resource was not found', status: 404
```

Comment:
If instead of rescuing directly here you let the exception flow out but use the `rescue_from_unless_local` at the top of this class, we can make sure to have standard error JSON and also let these errors explode in local development

Reply:
thought we'd want local behaving like production here....

Reply:
talking about how Rails typically shows exceptions in development instead of rescuing them and showing error pages